src/analyses/variable-sensitivity/abstract_environment.h

```diff
@@ -170,6 +170,19 @@ class abstract_environmentt
     const exprt &e,
     const namespacet &ns) const;
 
+  /// Wraps an existing object in any configured context object
+  ///
+  /// \param abstract_object: The object to be wrapped
+  ///
+  /// \return The wrapped abstract object
+  ///
+  /// Look at the configuration context dependency, and constructs
+  /// the appropriate wrapper object around the supplied object
+  /// If no such configuration is enabled, the supplied object will be
+  /// returned unchanged
+  virtual abstract_object_pointert
+  add_object_context(const abstract_object_pointert &abstract_object) const;
```

Comment:
I have a feeling I know what this does but it is not 100% clear to me how it fits in in the work-flow.